core-js/src/main/javascript/cdf-legacy/components/input.js

```diff
@@ -579,38 +611,38 @@ var TextareaInputComponent = BaseComponent.extend({
 
     var el = $("#" + name);
 
-    el.change(function() {
-      if(Dashboards.getParameterValue(myself.parameter) !== el.val()) {
+    el.change(function () {
+      if (Dashboards.getParameterValue(myself.parameter) !== el.val()) {
         Dashboards.processChange(name);
       }
     });
   },
-  getValue : function() {
+  getValue: function () {
     return $("#" + this.name).val();
   }
 });
 
 
 // Start by setting a sane i18n default to datepicker
 //TODO: move this to where we know for sure datepicker is loaded..
-if($.datepicker) {
-  $(function(){$.datepicker.setDefaults($.datepicker.regional[''])});
+if ($.datepicker) {
+  $(function () { $.datepicker.setDefaults($.datepicker.regional['']) });
 }
 
 var DateInputComponent = BaseComponent.extend({
-  update: function() {
+  update: function () {
     var myself = this;
-    var format = (myself.dateFormat == undefined || myself.dateFormat == null)? 'yy-mm-dd' : myself.dateFormat;
+    var format = (myself.dateFormat == undefined || myself.dateFormat == null) ? 'yy-mm-dd' : myself.dateFormat;
```

Comment:
redundant
`myself.dateFormat == undefined` will be true if its `undefined` or `null`